server/api/routes/repos.route.js

```diff
@@ -306,18 +307,24 @@ router
       .then(data => res.send(data))
       .catch(next);
   })
-  .get('/:repositoryId/forks', (req, res, next) => {
+  .get('/:repositoryId/forks', async (req, res, next)  => {
```

Comment:
```suggestion
  .get('/:repositoryId/forks', (req, res, next)  => {
```
async with no awaits